bridge/src/test/java/io/stargate/bridge/service/SchemaOperationsTest.java

```diff
@@ -253,4 +256,30 @@ public void schemaTableWithMaterializedView() {
     assertThat(response.getMaterializedViews(0).getColumnsCount() == 1).isTrue();
     assertThat(response.getMaterializedViews(0).getColumns(0).getName().equals("c")).isTrue();
   }
+
+  @Test
+  @DisplayName("Handle and propagate server-side NPE appropriately")
+  public void schemaTableWithServerSideNPE() {
+    // Given
+    StargateBridgeGrpc.StargateBridgeBlockingStub stub = makeBlockingStub();
+    when(persistence.decorateKeyspaceName(any(String.class), any())).thenReturn("my_stuff");
+
+    // Only need minimal schema for bootstrapping
+    io.stargate.db.schema.Schema schema = io.stargate.db.schema.Schema.build().build();
```

Comment:
Codacy complains here and I guess is correct this time:

> io.stargate.db.schema.Schema schema = io.stargate.db.schema.Schema.build().build();

Reply:
> Codacy complains here and I guess is correct this time:
> 
> > io.stargate.db.schema.Schema schema = io.stargate.db.schema.Schema.build().build();

Ah. Yes, will remove. Copy-paste flag. For once Codacy was right :)